compaction/compaction_task.hh

```diff
@@ -10,6 +10,28 @@
 
 #include "tasks/task_manager.hh"
 
+class compaction_task_impl : public tasks::task_manager::task::impl {
+public:
+    compaction_task_impl(tasks::task_manager::module_ptr module,
+            tasks::task_id id,
+            unsigned sequence_number,
+            std::string keyspace,
+            std::string table,
+            std::string entity,
```

Comment:
All current implementations pass empty string for this `entity` parameter. Is that really needed? Will there be a future task type which passes something here?